extensions/sql-format/package.json

```diff
@@ -0,0 +1,103 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "sql-format",
+  "title": "Format SQL",
+  "description": "Format a SQL cluase use standard SQL format",
+  "icon": "extension-icon.png",
+  "author": "tonngw",
```

Comment:
```suggestion
  "author": "tonngw",
  "contributors": ["stelo"],
```

Reply:
I have added it, happy working with you.